ui/src/telegrafs/reducers/index.ts

```diff
@@ -30,7 +29,18 @@ export const telegrafsReducer = (
         draftState.status = status
 
         if (list) {
-          draftState.list = list
+          draftState.list = list.map(l => {
```

Comment:
should this be `l`? i generally dislike single letter variable names on principle, but even in this case, i think `l` is the wrong abbreviation. it's not a `list` or `list item`, it's a telegraf config, right? you're looping over a list of telegraf configs? should it be `t`?

either way, i think naming the current iteration item, even though it's a small anonymous function, goes a long way in documenting the intent of our code